scripts/fill-messages-to-influx.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+var path = require('path'),
+    chalk = require('chalk'),
+    async = require('async'),
+    config = require(path.resolve('./config/config')),
+    configMongoose = require(path.resolve('./config/lib/mongoose')),
+    mongoose = require('mongoose'),
+    fluxIn = require(path.resolve('./modules/messages/server/services/message-to-influx.server.service')),
```

Comment:
`fluxIn` seems like a confusing variable name for the `message-to-influx.server.service`.